app/scripts/headerbar/headerbar-partial.html

```diff
@@ -0,0 +1,18 @@
+<div>
+    <a ui-sref="charts"><div id="logo"></div></a>
+    <ul class="nav nav-pills pull-right">
+
+        <mos-year-selector></mos-year-selector>
+
+        <li ui-sref-active="active"><a ui-sref="charts({year: ctl.year})"><div class="nav-icon nav-chart"></div>Charts</a></li>
+        <!-- TODO: Figure out how to show map highlight when detail/compare is active? -->
```

Comment:
An old TODO! Looks like this was copied from the old HTML. Is it still valid?

Reply:
I don't even know what it means. Shall I make a mystery issue?